code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -246,6 +246,21 @@
 	if(reagents.has_reagent("teslium"))
 		msg += "[t_He] is emitting a gentle blue glow!\n"
 
+	if(drunkenness && !skipface && stat != DEAD) //Drunkenness

```

Comment:
Do we really not have a better way to add to examines than a series of endless if statements?